test-apps/README.md

```diff
@@ -0,0 +1,44 @@
+gcloud-java Test Applications
+=============================
+
+This folder contains examples of end-to-end test apps for gcloud-java.
```

Comment:
I would not call it end-to-end test yet (though we can say that it could be used to end-to-end test).